### PR TITLE
Dev to Main

### PR DESCRIPTION
**Pull request by Vahan11**
Please, review.

### CODE REVIEW

File: assets/js/registration.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+let timeZone = document.querySelectorAll(".num");
```
Comment (vre2h) [inline assets/js/registration.js]: `num` is not good class name, it's hard to understand what it means

File: assets/js/registration.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+let countDownDate = new Date("July 01, 2021 00:00:00").getTime();
```
Comment (vre2h) [inline assets/js/registration.js]: Why July 1 and not `new Date()`

File: assets/js/registration.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+let interval = setInterval(() => {
```
Comment (vre2h) [inline assets/js/registration.js]: `interval` is not good variable name, `intervalId` is better one.

File: index.html [hunk 4]
```diff
@@ -285,6 +379,70 @@
...
+            Find more information about the ISA and other organizational details in the F.A.Q.<br>
+            section.
+          </div>
+        </div>
+        
+      </div>
+    </div>
+
+    <!-- Consultation -->
+    <div id="cons" class="cons">
```
Comment (vre2h) [inline index.html]: `cons` is not a good variable name. Cons mean disadvantage, `consultation` is better one

**Review by vre2h — CHANGES_REQUESTED** [review]

File: assets/js/registration.js (hunk 2 above)
Comment (Vahan11) [reply assets/js/registration.js]: Here you can write any date for the countdown.  Now updated to 21st July :)

File: assets/js/registration.js (hunk 1 above)
Comment (Vahan11) [reply assets/js/registration.js]: fixed

File: assets/js/registration.js (hunk 3 above)
Comment (Vahan11) [reply assets/js/registration.js]: fixed

File: index.html (hunk 4 above)
Comment (Vahan11) [reply index.html]: fixed

**Review by vre2h — APPROVED** [review]

**Pull request merged**